core-tests/shared/src/test/scala/zio/ZRefSynchronizedSpec.scala

```diff
@@ -490,7 +490,7 @@ object ZRefMSpec extends ZIOBaseSpec {
           tuple   <- composed.get
           (a, b)   = tuple
         } yield assert(a)(equalTo(6765)) && assert(b)(equalTo(10946))
-      },
+      } @@ nonFlaky,
```

Comment:
Is this strictly necessary?

Reply:
Wanted to confirm that the updating of the two `Ref` values is performed atomically and we didn't just get "lucky" that two of the fibers updating the `Ref` didn't make conflicting updates, but happy to remove if you want.